src/kaczmarz/_utils.py

```diff
@@ -0,0 +1,20 @@
+from scipy import sparse
+
+
+def scale_rows(A, v):
+    """Scale each row of A by the corresponding entry of v and return the
+    result."""
```

Comment:
Squeeze this onto a single line even if it overflows unless the linter complains.